spec/services/activities/import_from_csv_spec.rb

```diff
@@ -273,8 +299,6 @@
       expect(new_activity.call_close_date).to eq(DateTime.parse(new_activity_attributes["Call close date"]))
       expect(new_activity.call_present).to eq(true)
       expect(new_activity.planned_start_date).to eq(DateTime.parse(new_activity_attributes["Planned start date"]))
-      expect(new_activity.planned_end_date).to eq(DateTime.parse(new_activity_attributes["Planned end date"]))
-      expect(new_activity.actual_start_date).to eq(DateTime.parse(new_activity_attributes["Actual start date"]))
```

Comment:
Are these two deleted lines intentional?

Reply:
No! Thank you! How did I not see those. I'll go and have a look.

Reply:
Must have been a mec+vim typo!